src/tempprocessing.f90

```diff
@@ -2884,14 +2898,19 @@ real(dp) function Bnormalized(TheDaysToCCini, TheGDDaysToCCini,&
      end if
 
      ! 5. Calculate Bnormalized
+     i = 0
      do Dayi = 1, L1234
          ! 5.1 growing degrees for dayi
          if (GetTemperatureFile() == '(None)') then
              GDDi = DegreesDay(Tbase, Tupper, TDayMin, TDayMax,&
                                GetSimulParam_GDDMethod())
          elseif (GetTemperatureFile() == '(External)') then 
-             Tndayi = real(GetTminRun_i(GetCrop_Day1()-GetSimulation_FromDayNr()+Dayi),kind=dp)
-             Txdayi = real(GetTmaxRun_i(GetCrop_Day1()-GetSimulation_FromDayNr()+Dayi),kind=dp)
+             i = i + 1
+             if (i == size(GetTminCropReferenceRun())) then
+                 i = 1
+             end if
+             Tndayi = real(GetTminCropReferenceRun_i(i),kind=dp)
+             Txdayi = real(GetTmaxCropReferenceRun_i(i),kind=dp)
              GDDi = DegreesDay(Tbase, Tupper, Tndayi, Txdayi, &
```

Comment:
According to Dirk's implementation, it's indeed the real temperature that's used here, and not the Reference. The Reference is used only in SumCalendarDaysReferenceTnx. To be double-checked with Dirk what should be used here.

Reply:
I understand that the Reference is read. See new line 2833:
            open(newunit=fTemp, file=trim(GetPathNameSimul()//'TCropReference.SIM'), &

Reply:
Good catch! You are right. My bad. So my preliminary implementation of the External if section was wrong!